Source/Android/app/src/main/res/layout-ldrtl/list_item_setting.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout android:id="@+id/root"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="?android:attr/selectableItemBackground"
+    android:clickable="true"
+    android:focusable="true"
+    android:minHeight="72dp"
+    android:nextFocusLeft="@id/button_more_settings">
+
+    <TextView
+        android:id="@+id/text_setting_name"
+        style="@style/TextAppearance.MaterialComponents.Headline5"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentStart="true"
+        android:layout_alignParentTop="true"
+        android:layout_marginEnd="@dimen/spacing_large"
+        android:layout_marginStart="@dimen/spacing_large"
+        android:layout_marginTop="@dimen/spacing_large"
+        android:textSize="16sp"
+        android:textAlignment="viewStart"
+        tools:text="Setting Name" />
+
+    <TextView
+        android:id="@+id/text_setting_description"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentStart="true"
+        android:layout_alignStart="@+id/text_setting_name"
+        android:layout_below="@+id/text_setting_name"
+        android:layout_marginBottom="@dimen/spacing_large"
+        android:layout_marginEnd="@dimen/spacing_large"
+        android:layout_marginStart="@dimen/spacing_large"
+        android:layout_marginTop="@dimen/spacing_small"
+        android:textAlignment="viewStart"
+        tools:text="@string/overclock_enable_description" />
```

Comment:
```suggestion
    <TextView
        android:id="@+id/text_setting_name"
        style="@style/TextAppearance.MaterialComponents.Headline5"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:layout_alignParentStart="true"
        android:layout_alignParentTop="true"
        android:layout_marginEnd="@dimen/spacing_large"
        android:layout_marginStart="@dimen/spacing_large"
        android:layout_marginTop="@dimen/spacing_large"
        android:textSize="16sp"
        android:textAlignment="viewStart"
        android:layout_toStartOf="@+id/button_more_settings"
        tools:text="Setting Name" />

    <TextView
        android:id="@+id/text_setting_description"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:layout_alignParentStart="true"
        android:layout_alignStart="@+id/text_setting_name"
        android:layout_below="@+id/text_setting_name"
        android:layout_marginBottom="@dimen/spacing_large"
        android:layout_marginEnd="@dimen/spacing_large"
        android:layout_marginStart="@dimen/spacing_large"
        android:layout_marginTop="@dimen/spacing_small"
        android:layout_toStartOf="@+id/button_more_settings"
        android:textAlignment="viewStart"
        tools:text="@string/overclock_enable_description" />
```

GitHub won't let me suggest this in the ltr layout file but you should apply it similarly there too.

This view would normally have the text overlap the button when long enough. But conveniently, setting match_parent as the width and ending the text view at the start of the button makes sure we avoid the button as the screen width gets smaller or as the text gets long enough. Setting the view as match_parent has the added benefit of making sure the text view bounds don't get crushed when we set the more settings button as gone.

Once again this also makes me sad that there isn't an option for `nextFocusStart` so we wouldn't even have to make this file...